### PR TITLE
Langing page

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created landing page 

landingpage
landingpage2

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  text-decoration: none;
+}
+
+.btn-navbar-doc:hover {
+  background-color: var(--primary-050);
+  color: var(--primary-900);
+}
+
+.navbar-brand-doc {
+  height: 45px;
```
Comment (aryan-13) [inline styles.css]: Rems can be used here

**Review by aryan-13 — COMMENTED** [review]
The code looks clean! Great work!

**Review by ZuberDunge — APPROVED** [review]
Looks good to me. Also landing page is awesome, good work.

File: styles.css (hunk 1 above)
Comment (prachi-sahani) [reply styles.css]: Done

**Pull request merged**